libcst/codemod/visitors/_gather_imports.py

```diff
@@ -114,3 +120,9 @@ def visit_ImportFrom(self, node: libcst.ImportFrom) -> None:
                     return
 
                 self.object_mapping[module].update(new_objects)
+            for ia in nodenames:
```

Comment:
It's probably better to use a longer name for `ia` here as well (imported_attribute I'm guessing?)